coast/DOMAIN.py

```diff
@@ -27,7 +27,7 @@ def set_command_variables(self):
                 e2u, e2v, e2t, e2f (t,y,x) - double - (m)
         """
         try:
-            self.bathy_metry = self.dataset.bath_metry
+            self.bathy_metry = self.dataset.bathy_metry
```

Comment:
Isn't bathymetry a single word?

Reply:
yes, however, the users would like variable names to stay the same, to allow smooth adoption of the package.